scripts/run_test_in_devicefarm.sh

```diff
@@ -47,11 +42,57 @@ curl -H "Content-Type:application/octet-stream" -T $full_path $app_package_url
 echo "Waiting for uploads to complete"
 sleep 10
 
+# Get oldest device we can test against.
+minDevice=$(aws devicefarm list-devices \
+                --region="us-west-2" \
+                --filters '[
+                    {"attribute":"AVAILABILITY","operator":"EQUALS","values":["HIGHLY_AVAILABLE"]},
+                    {"attribute":"PLATFORM","operator":"EQUALS","values":["ANDROID"]},
+                    {"attribute":"OS_VERSION","operator":"GREATER_THAN_OR_EQUALS","values":["7"]},
+                    {"attribute":"OS_VERSION","operator":"LESS_THAN","values":["8"]},
```

Comment:
Could the LESS_THAN = 7.1 so it run on API 24 which is our min supported version in dev-preview?